target/linux/ath79/dts/ar9342_ubnt_nanostationacl.dts

```diff
@@ -0,0 +1,40 @@
+// SPDX-License-Identifier: GPL-2.0
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "ar9342_ubnt_wa.dtsi"
+
+/ {
+	compatible = "ubnt,nanostation-acl", "ubnt,wa";
```

Comment:
ubnt,nanostation-ac-loco please. -acl is confusing and not that obvious to match to ac-loco

Reply:
That's right. I'll fix that right away